pages/product.css

```diff
@@ -0,0 +1,201 @@
+@import url("https://fonts.googleapis.com/css2?family=Comfortaa:wght@700&family=Quicksand:wght@300&family=Roboto:wght@100;300;400;500&display=swap");
+
+:root {
+  --primary-color: #fff;
+
+  --secondary-color: #4b5563;
+  /* --action-color: #f26430;confused */
+  --action-color: #ff715b;
+
+  --background-color: #171717;
+}
```

Comment:
You can add these colors in the style guide or any other file and then import them into every CSS file.